apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -129,10 +136,10 @@ function DescriptionCard(props) {
 DescriptionCard.propTypes = {
   id: PropTypes.string,
   title: PropTypes.string.isRequired,
-  href: PropTypes.string.isRequired,
-  imgSrc: PropTypes.string.isRequired,
+  href: PropTypes.string,
+  imgSrc: PropTypes.string,
```

Comment:
This may be out of scope - but it looks like the reason some of these props are being marked as not required is because we're trying to use the `DescriptionCard` component as a kind of header component, right? I wonder if it would be worth creating a separate header card component or something that only had a title and a description (no image) for clarity's sake